### ISSUE TITLE
Keyboad events

### ISSUE THREAD

**Issue by ok1cdj** [tomvit/ja2mqtt#18]
Hello, is possible to also events from keyboard like access was granted to the person or bad password entered ?

Thanks.

**Comment by tomvit** [comment]
I do not think so this is possible. ja2mqtt can only capture events provided by the JA-121T module.
